src/deployment/android.md

```diff
@@ -355,7 +355,7 @@ to verify that the values are correct.
   a reference to a string resource.
 
 `buildToolsVersion`
-: If you're using Android plugin for Gradle 3.0.0 or higher,
+: If you're using [Android plugin for Gradle][] 3.0.0 or higher,
```

Comment:
Since you're here, consider dropping the "If ..." part of this sentence. All versions of Android Studio that we support [require 3.2 or greater](https://developer.android.com/build/releases/gradle-plugin#android_gradle_plugin_and_android_studio_compatibility).